e2e/upgrade-testing-rbd.go

```diff
@@ -0,0 +1,197 @@
+package e2e
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo" // nolint
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
+)
+
+const (
+	pvcSize       = "1Gi"
+	pvcExpandSize = "10Gi"
+)
+
+var (
+	pvc *v1.PersistentVolumeClaim
```

Comment:
avoid defining global variables and const

Reply:
Since I am using same PVC for two separate tests, the global variable for required here.

Reply:
add it inside describe not as global

Reply:
Okay, thanks.